test/continuous.jl

```diff
@@ -52,4 +52,15 @@ using ForwardDiff
     @test quantile(d, 1.0) == +Inf
 
     @test rand(d) == 0.5
-end    
+end
+
+# Test for parameters beyond those supported in R references
+@testset "VonMises with large kappa" begin
+    d = VonMises(1.1, 1000)
+    @test var(d) ≈ 0.0005001251251957198
+    @test entropy(d) ≈ -2.034688918525470
+    @test isapprox(cf(d, 2.5), -0.921417 + 0.38047im, atol=1e-6)
```

Comment:
you can write this as 
```
@test cf(d, 2.5) ≈ -0.921417 + 0.38047im atol=1e-6
```
and the `@test` macro will rewrite it appropriately.

Reply:
I don't know that. I updated those in the latest commit.